source/validate_id.cpp

```diff
@@ -1237,7 +1232,10 @@ bool idUsage::isValid<SpvOpAccessChain>(const spv_instruction_t* inst,
         << num_indexes_limit << ". Found " << num_indexes << " indexes.";
     return false;
   }
-
+  if (num_indexes <= 0) {
```

Comment:
This works only because we rely on the binary parser to validate that at least we have 4 words in the instruction.
Otherwise this code would be wrong because num_indexes is a size_t and is therefor unsigned.
The safer pattern in general is to check inst->words.size() >= 4.

But this code works, subtlly.